src/test/regress/sql/pg15.sql

```diff
@@ -976,3 +976,115 @@ SET client_min_messages TO ERROR;
 DROP SCHEMA pg15 CASCADE;
 DROP ROLE rls_tenant_1;
 DROP ROLE rls_tenant_2;
+
+
+-- create/drop database for pg > 15
+
+
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_1_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts4'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :worker_2_port
+\set create_drop_db_tablespace :abs_srcdir '/tmp_check/ts5'
+CREATE TABLESPACE create_drop_db_tablespace LOCATION :'create_drop_db_tablespace';
+
+\c - - - :master_port
+create user create_drop_db_test_user;
+set citus.enable_create_database_propagation=on;
+CREATE DATABASE mydatabase
+    WITH TEMPLATE = 'template0'
+            OWNER = create_drop_db_test_user
+            CONNECTION LIMIT = 10
+            ENCODING = 'UTF8'
+            STRATEGY = 'wal_log'
+            LOCALE = ''
+            LC_COLLATE = 'POSIX'
+            LC_CTYPE = 'POSIX'
+            ICU_LOCALE = 'und'
+            LOCALE_PROVIDER = 'icu'
+            COLLATION_VERSION = '1.0'
+            TABLESPACE = create_drop_db_tablespace
+            ALLOW_CONNECTIONS = true
+            IS_TEMPLATE = false
+            OID = 966345;
+
+SELECT pd.datname, pd.encoding, pd.datlocprovider,
```

Comment:
Can we also print pd.oid for pg >= 15 tests?

Reply:
For the cases where "oid" is specified in CREATE DATABASE; we should not query "oid" otherwise as it might get flaky over time.

Reply:
> For the cases where "oid" is specified in CREATE DATABASE; we should not query "oid" otherwise as it might get flaky over time.

I meant this one, sorry:

For the cases where "oid" is **NOT** specified in CREATE DATABASE; we should not query "oid" otherwise as it might get flaky over time.

In summary, I meant printing oid when it's specified in CREATE DATABASE command and not doing so otherwise.